.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
```

Comment:
This was intentionally removed in master. You're reintroducing husky here.

Reply:
Oh FFS. I only wanted to delete a chuffing CSS file.